app/wasm/test/custom_query_test.go

```diff
@@ -86,6 +87,71 @@ func TestQueryPool(t *testing.T) {
 	assertValidShares(t, resp.Shares, atomPool)
 }
 
+func TestQuerySpotPrice(t *testing.T) {
```

Comment:
Good test.
Let's turn this into a table test in a follow up PR (when adding routes and all)